server/docker-entrypoint.sh

```diff
@@ -1,26 +1,13 @@
 #!/bin/sh
 
-# uninstall the current bcrypt modules
-# npm uninstall bcrypt
-
-# install the bcrypt modules for the machine
-# This is needed because bcrypt uses different modules for different architectures.
-# If the node_modules folder is shared the process will crash otherwise.
-# https://medium.com/hacktive-devs/the-bcrypt-bg-on-docker-9bc36cc7f684
-
-# npm install bcrypt
-
-# The reinstall of bcrypt has become for the time being obsolet due to the fact that the dependency was replaced by bcryptjs
-# TODO Evaluate if the usage of bcryptjs is sufficent enough
-
 echo "Waiting for DB to start..."
-./wait-for db:5432 
+./wait-for db:5432 #TODO: Maybe better wait in the app where the DB is needed
```

Comment:
Q: Diese Funktion habe ich tatsächlich damals ins Backend eingebaut. Finde es aber hier persöblich besser aufgehoben. Was spricht hier dagegen, bzw. für das Warten im server?

Reply:
Hmm, ist vermutlich hier im Projekt egal und Geschmackssache.
Prinzipiell sollten services natürlich entkoppelt funktionieren und den failover zur Laufzeit machen. Bei einer größeren Applikation mit mehreren Webserver Instanzen wird das interessanter.

Könnt Ihr also auch gerne so lassen 😊

Nur dran denken, dass der entrypoint in docker die signale vom system bekommt und die kindprozesse entsprechend steuern muss. Aber dafür ist das exec ja jetzt da